setup.py

```diff
@@ -97,7 +97,7 @@
     'pyyaml==3.11',
     'requests==2.9.1',
     'scikit-learn==0.17.1',
-    'ipykernel==4.4.1',
+    'ipykernel<=4.5.2',
```

Comment:
I don't think I want to leave it ambiguous as to which version we install.

Can you explain what goes wrong if this is just `ipykernel==4.5.2`?